src/App.tsx

```diff
@@ -50,15 +44,15 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && loaded && <Loader />}
```

Comment:
The condition for rendering the `Loader` component seems incorrect. It should check if the posts are being loaded, not if they are already loaded. Consider changing the condition to `author && !loaded` to reflect the loading state correctly.